nixos/modules/services/misc/ollama.nix

```diff
@@ -0,0 +1,41 @@
+{ config, lib, pkgs, ... }: let
+
+  cfg = config.services.ollama;
+
+in {
+
+  options = {
+    services.ollama = {
+      enable = lib.mkEnableOption (
+        lib.mdDoc "Server for local large language models"
+      );
```

Comment:
I would like `package = mkPackageOption pkgs "ollama" { };`.

Reply:
Thank you, added